nncf/config_schema.py

```diff
@@ -354,6 +363,8 @@ def with_attributes(schema: Dict, **kwargs) -> Dict:
         "algorithm": {
             "const": QUANTIZATION_ALGO_NAME_IN_CONFIG
         },
+        "name": with_attributes(_STRING,
```

Comment:
> @RikAllen, could you provide documentation for the block floating point quantization approach? I mean algorithm description and so on.

I think it would be great to have a separate Readme file that explains the method, flow (path to the inference) and claims any results from the accuracy prospective at least.

Reply:
Added blockFloatingPoint.md with a description. We will add results - in your normal situation those seem to belong in other places along with sparsity results etc.